packages/zk-certificates/scripts/checkZkCertificateRegistry.ts

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ethers } from 'hardhat';
+
+/**
+ * Performs some batch mint of SBTs.
+ */
+async function main() {
+  // parameters for test interaction
+  // const SBTAddress = '0xC1d40362A2e6c295DC53BAc79F65F81D030Da3Bd';
+  // const SBTAddress = '0x75C47b5210658C6beFdDb23ab34B0B025979978e';
+  // const SBTAddress = '0x589A65F4434c63A220DEDb050C3Cc03ee43fBF13';
```

Comment:
These comments do not make sense in this file.

Reply:
removed.